content/influxdb/v2.3/write-data/troubleshoot.md

```diff
@@ -321,4 +322,33 @@ InfluxDB logs the following `rejected_points` entry to the `_monitoring` bucket:
 |:------------------|:-------|:-------|:-------------------|:--------------|:---------|:------------|:----------------------------------|:---------|
 | rejected_points   | count  | 1      | a7d5558b880a93da   | temperature   | String   | airSensors  | field type mismatch with schema   | Float    |
 
-{{% /cloud-only %}}
\ No newline at end of file
+{{% cloud-only %}}
+
+## Troubleshoot MQTT ingest errors
+
+If you're having issues ingesting [MQTT data](/influxdb/v2.3/write-data/no-code/load-data/#set-up-a-cloud-native-subscription/) into InfluxDB, try the following:
```

Comment:
Should this URL be v2.3 or cloud? Maybe it doesn't really matter inside cloud-only shortcode.

Reply:
@sanderson can you weigh in?